atlasdb-workload-server-antithesis/var/docker-compose.yml

```diff
@@ -57,7 +57,9 @@ services:
 
   timelock1.palantir.pt:
     image: palantirtechnologies/timelock-server-distribution:unspecified
-    command: [ bash, -c, 'cp var/conf/timelock.distributed.yml var/conf/timelock.yml  
+    environment:
+      HOSTNAME: timelock1.palantir.pt:8421
+    command: [ bash, -c, 'sed "s/{{HOSTNAME}}/$${HOSTNAME}/g" var/conf/timelock.distributed.yml > var/conf/timelock.yml
```

Comment:
We have to use the weird `$${VAR}` syntax in order to be able to use the envrionment variables in the docker-compose file itself. Source: https://stackoverflow.com/questions/40447029/docker-compose-environment-variable-for-command